app/views/playlists/_item_list.html.erb

```diff
@@ -34,14 +34,23 @@ Unless required by applicable law or agreed to in writing, software distributed
 </div>
 
 <div class="row">
-  <div class="playlist-autoplay-wrapper">
+  <div class="playlist-autoplay-wrapper col-sm-6">
     <span class="playlist-autoplay-label">Autoplay</span>
     <input type="checkbox" <%= 'checked' if params['autoadvance'].nil? or params['autoadvance']=='true' %>
       data-toggle="toggle" data-size="mini" name="autoadvance" />
   </div>
-  <div class="text-right">
+  <div class="col-sm-6">
     <%= render 'share' if will_partial_list_render? :share %>
   </div>
+  <div id="share-list" class="mt-3 collapse">
+    <ul class="nav nav-tabs share-tabs">
+      <%= render_conditional_partials :share, section:'share-tabs' %>
+    </ul>
+
+    <div class="tab-content">
+      <%= render_conditional_partials :share, section:'tab-content' %>
+    </div>
+  </div>
```

Comment:
I see the `mt-3` class was added to this `<div>`, but I'm curious why did you move this div out of the share partial?

Reply:
Share partial is rendered within the div with the `col-*` class, which spans the share panel to the width of that space. But it should span the entire width of the row under the share and autoplay buttons.
Do you think we need to render it from a partial instead of putting it in here, because of the condition?

Reply:
Now that you mention it, yes!  :slightly_smiling_face: 
I was thinking that semantically they fit better in the share partial since there is JS in that partial which references them.
So how about pushing the wrapping div into the partial instead of moving the other div out of the partial?  Would that work?

Reply:
Yes that would work! Let me try that :+1: